functions/v2/index.js

```diff
@@ -68,8 +69,14 @@ exports.helloAuditLog = cloudevent => {
   console.log('Subject:', cloudevent.subject);
 
   // Print out details from the Cloud Audit Logging entry
-  const payload = cloudevent.data.protoPayload;
-  console.log('Principal:', payload.authenticationInfo.principalEmail);
+  const metadata =
+    cloudevent.data &&
+    cloudevent.data.protoPayload &&
+    cloudevent.data.protoPayload.requestMetadata;
+  if (metadata) {
+    console.log('Caller IP:', metadata.callerIp);
+    console.log('User Agent:', metadata.callerSuppliedUserAgent);
```

Comment:
nit: these might not be interesting either. Do we have an interesting use case? What is usually in the protoPayload?

Reply:
Resolved over chat - we decided to show some other properties, too.